lib/pbench/cli/server/shell.py

```diff
@@ -61,7 +59,9 @@ def keycloak_connection(
         logger: logger
     Returns:
         0 if successful
+        > 0 if unsuccessful
     """
+    return_val = 1
```

Comment:
How pessimistic!  😉 

Seriously, though -- if we fail, I don't think that the caller is going to care whether we failed on the first try or the fifth, so there's no point in having `return_val` be a variable:  just return zero on success and `1` on failure and let the code be simpler (which, in turn, lets the documentation be simpler...well, it _would_ be, if you actually described what the non-zero values mean.... 😉).  And, if someone wants to know how many failures there were, they can look in the log.